src/SpacetimeDBClient.cs

```diff
@@ -1101,6 +875,33 @@ void IDbConnection.Unsubscribe(QueryId queryId)
         void IDbConnection.AddOnDisconnect(WebSocket.CloseEventHandler cb) => webSocket.OnClose += cb;
     }
 
+    internal struct ParsedDatabaseUpdate
```

Comment:
Doc comment here, explaining that a parsed database update contains updates for all tables, constructor weirdness, etc